controller/articles.controllers.js

```diff
@@ -0,0 +1,18 @@
+const selectArticleById = require("../model/articles.models");
+
+function getArticlesById(request, response, next) {
+  const articleId = request.params.article_id;
+
+  selectArticleById(articleId)
+    .then((article) => {
+      if (article.rows.length === 0) {
```

Comment:
this `Promise.reject` should be done in the model. You should ideally be sending the `rows[0]` back from the model as well